jetty-util/src/main/java/org/eclipse/jetty/util/thread/AutoLock.java

```diff
@@ -36,6 +35,8 @@
  */
 public class AutoLock implements AutoCloseable, Serializable
 {
+    private static final long serialVersionUID = 3300696774541816341L;
```

Comment:
We should make this change for all of our Serializable classes.

https://product.hubspot.com/blog/java-11-dont-rely-on-generated-serialversionuid

Reply:
Opened as Issue #5100